code/datums/id_trim/jobs.dm

```diff
@@ -831,24 +831,11 @@
 	subdepartment_color = COLOR_MEDICAL_BLUE
 	sechud_icon_state = SECHUD_PARAMEDIC
 	minimal_access = list(
-		ACCESS_MAINT_TUNNELS,
-		ACCESS_MECH_MEDICAL,
+		ACCESS_EXTERNAL_AIRLOCKS,
 		ACCESS_MEDICAL,
+		ACCESS_MAINT_TUNNELS,
 		ACCESS_MORGUE,
-		)
-	extra_access = list(
-		ACCESS_BIT_DEN,
-		ACCESS_CARGO,
-		ACCESS_CONSTRUCTION,
-		ACCESS_HYDROPONICS,
-		ACCESS_SURGERY,
-		ACCESS_VIROLOGY,
-		ACCESS_PHARMACY,
-		ACCESS_MINERAL_STOREROOM,
-		ACCESS_MINING,
-		ACCESS_MINING_STATION,
-		ACCESS_SCIENCE,
-		ACCESS_SERVICE,
```

Comment:
1) You're removing the additional access within medical itself on lowpop, which is a separate issue to the problem of expanded access. They should keep their expanded lowpop medical access. This access had nothing to do with needing to access areas in an emergency and more to do with the fact that lowpop forces people within a department to do perform broader duties.

2) There is an argument to be made that this new system does not function as well in low pop situations for specifically paramedics. It might not be the worst thing for them to keep broad access during lowpop even with this new system.

Reply:
Well, lowpop is when you're less likely to see people in a department as well as have an AI/silicons present to help with access. So while it might enable them to roam through departments more, the possibility of other roles needing to enter another department for things is more understandable in those circumstances as well. Paramedics are less likely to be shoving aside people in a department if there is a significant lack of people in the first place.